packages/core/package.json

```diff
@@ -18,20 +18,43 @@
     "functional"
   ],
   "ava": {
+    "babel": {
+      "plugins": [
+        [
+          "transform-object-rest-spread",
```

Comment:
I thought so, too. But it seems quite a lot of features that once were part of the es2017 preset have been removed from it recently. Object-spread isn't included anymore... :-/

https://babeljs.io/docs/plugins/preset-es2017/